app/models/holiday.rb

```diff
@@ -18,6 +18,10 @@ def self.merged_for_calendar_view
       .order(:start_at)
   end
 
+  def self.scoped_to_user(user)
+    where(user_id: [user.id, nil])
```

Comment:
I don't really like that it's a bit confusing at first and rubocop would complain about the dynamic assignment. I changed the method name and rewrote it to be more obvious.